static/css/components/librarian-dashboard.less

```diff
@@ -0,0 +1,59 @@
+details.librarian-dashboard {
+  color: @grey;
+  margin-bottom: 15px;
+
+  summary {
+    font-weight: 700;
+  }
```

Comment:
In context of the page, it's become a bit bold-heavy at the top; so I think this can be a bit more muted.
```suggestion
```